app/models/patient.rb

```diff
@@ -46,8 +48,46 @@ class Patient < ApplicationRecord
                                                          'No Identified Risk',
                                                          nil, ''] }
 
+  %i[address_state
+     ethnicity
+     monitored_address_state
+     preferred_contact_method
+     preferred_contact_time
+     sex].each do |enum_field|
+    validates enum_field, on: :api, inclusion: {
+      in: VALID_ENUMS[enum_field],
+      message: "%<value>s is not an acceptable value for '#{VALIDATION[enum_field][:label]}', acceptable values are: '#{VALID_ENUMS[enum_field].join("', '")}'"
+    }, allow_blank: true
+  end
+
+  %i[primary_telephone
+     secondary_telephone].each do |phone_field|
+    validates phone_field, on: :api, phone_number: true
+  end
+
+  %i[date_of_birth
+     last_date_of_exposure
+     symptom_onset].each do |date_field|
+    validates date_field, on: :api, date: true
+  end
+
+  %i[address_city
+     address_line_1
+     address_state
+     address_zip
+     date_of_birth
+     first_name
+     last_date_of_exposure
```

Comment:
Testing this locally and I realized `last_date_of_exposure` should not be required unless they are enrolling someone in the exposure workflow. Additionally, once we support continuous exposure, they can have continuous exposure set OR LDE for the exposure workflow. If they're enrolling someone in the isolation workflow, then symptom onset is what's required. I'm wondering if we want to potentially exclude those two fields as required for the API for now, as that seems like it could be a bit more complex.

Reply:
Since the API does support the `isolation` flag, how do you feel about a validator that checks if `isolation` is `true`, and if so requires `symptom_onset`. And then I would also remove the requiring of `last_date_of_exposure`, since you're right, it doesn't seem right to require `last_date_of_exposure` until we support `continuous_exposure` and can require one or the other.

Reply:
Ok stopped requiring `last_date_of_exposure` in 124bf1, and added a requirement of `symptom_onset` (only if `isolation` is `true`) in 8b66a63.

Reply:
Awesome, thanks Nick.